src/platforms/esp32/components/avm_sys/sys.c

```diff
@@ -546,13 +546,25 @@ term sys_get_info(Context *ctx, term key)
         term_set_map_assoc(ret, 3, globalcontext_make_atom(glb, revision_atom), term_from_int32(info.revision));
         return ret;
     }
-    if (key == globalcontext_make_atom(glb, esp_idf_version_atom)) {
```

Comment:
Sorry to be pedantic here on this PR, I didn't think about this.
We cannot remove features or break existing behavior in release-0.6 branch.
Let's keep it, and make a cleanup PR aimed to main after we merge this one